frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -255,3 +255,46 @@ Feature: Pipeline Runs
              Then user will see the pipeline run name with failed status
               And user will see failure message below pipeline runs
               And user will also see the log snippet
+
+
+        @regression
+        Scenario Outline: Start pipeline modal with PVC Workspaces
+            Given pipeline "pipeline-workspace" with PVC "git-PVC" workspace is present
+             When user selects "Start" option from kebab menu for pipeline "pipeline-workspace"
+              And user navigates to Workspaces section
+              And user clicks on dropdown with PVC selected by default to check options
+              And user clicks on dropdown with PVC names
+             Then user is able to see the PVC "git-PVC" and "create PVC" option present
+
+
+        @regression
+        Scenario Outline: Start pipeline modal with no Workspaces
+            Given pipeline "pipeline-no-workspace" with no workspace is present
+             When user selects "Start" option from kebab menu for pipeline "pipeline-no-workspace"
+              And user navigates to Workspaces section
+             Then user can see on dropdown with PVC selected by default
+              And user can see Create PVC selected
+
+
+        @regression
+        Scenario Outline: Show advanced PVC options
+            Given pipeline "pipeline-no-workspace" with no workspace is present
+             When user selects "Start" option from kebab menu for pipeline "pipeline-no-workspace"
+              And user navigates to Workspaces section
+              And user clicks Show advanced PVC options
+             Then user can see Storage Class
+              And user can see Persistent Volume Claim Name
+              And user can see Access Mode
+              And user can see Size
+              And user can see Volume Mode
+
+
+        @regression
+        Scenario Outline: Create PVC workspace in Start pipeline modal
+            Given pipeline "pipeline-no-workspace" with no workspace is present
+             When user selects "Start" option from kebab menu for pipeline "pipeline-no-workspace"
+              And user navigates to Workspaces section
+              And user clicks Show advanced PVC options
+              And user enters name "workspace-ex" in Persistent Volume Claim Name section
+              And user clicks on Start
+             Then user can see Workspace "workspace-ex" in pipeline details page
```

Comment:
As I mentioned before `volumeClaimTemplate` is the one we get the form on and there is no `name` field.

Also the form should have enough defaults to not need you to expand the options.